elasticsearch/utils/es_seed_acl

```diff
@@ -6,6 +6,13 @@ ES_CLUSTER_HOST=${ES_CLUSTER_HOST:-"localhost"}
 ES_CLUSTER_PORT=${ES_CLUSTER_PORT:-9300}
 MAX_WAIT_SEED=${MAX_WAIT_SEED:-$((60*60*24*7))} #one week
 
+NODES=$(es_util --query=_cat/nodes | wc -l)
+if [ $NODES -ge 3 ] ; then
```

Comment:
this should probably be `2`, right?

Reply:
I thought we said 3 but one replica for a two node cluster would be valid, though is it not bad practice to have a 2 node cluster because we would have 2 masters?

Reply:
thats for a forming a quorum of masters, completely different than having redundancy of shards